Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -181,7 +181,6 @@ void CommonAsmRoutines::GenMfcr()
   X64Reg cr_val = RSCRATCH_EXTRA;
   XOR(32, R(dst), R(dst));
   // we only need to zero the high bits of tmp once
```

Comment:
Now that comment doesn't make sense anymore. Can you put another one with the XOR to explain why we need it cleared all the time all of a sudden?